src/infra/nodemailer/setup.js

```diff
@@ -0,0 +1,41 @@
+import nodemailer from 'nodemailer';
+import * as dotenv from 'dotenv';
+import { generatingQRCode } from '../../utils/generateQRCode.js';
+
+dotenv.config();
+
+let transporter = nodemailer.createTransport({
+  host: process.env.email_host,
+  port: process.env.email_port,
+  secure: false,
+  service: process.env.email_service,
+  auth: {
+    user: process.env.email_user,
+    pass: process.env.email_pass
+  }
+});
+
+export async function sendEmail(studentData) {
+
+  const QRCode = await generatingQRCode();
+
+  var message = {
```

Comment:
Como `message` nunca muda, pode ser inicializada como const. Obs.: Evitar utilizar var pois pode causar problemas com escopos.

```suggestion
  const message = {
```

Reply:
Certo
Tinha feito como tu falou lá sakdhaskd, mas tudo bem